templates/terraform/cloudsql/main.tf

```diff
@@ -0,0 +1,82 @@
+# NOTE: This is needed because CloudSQL doesn't allow DB names to be reused
+# within the week that they're deleted, so whenever we want to tear down and
+# recreate the instance we need to generate a new name (vs waiting a week).
+resource random_id cloudsql_random_id {
+  byte_length = 8
+}
+
+# Create a single DB instance to back our applications.
+# NOTE: We might need multiple instances in prod, if it turns out there's too
+# much resource contention sharing a single instance.
+resource google_sql_database_instance postgres {
+  provider = google.target
+  depends_on = [var.dependencies]
```

Comment:
Need to put the `[ ]` brackets here otherwise terraform yells that you need a static list here.